From 57d7fc1fcad91a0411dbcc8b7c781e005012a233 Mon Sep 17 00:00:00 2001
From: Pavel Dubrova <pashadubrova@gmail.com>
Date: Wed, 26 Jan 2022 21:35:37 +0200
Subject: [PATCH 095/175] msm: ipa3: Remove barrier for IPAv3 for pipe
 suspend/resume

Remove barrier for IPA targets under IPAv4 for pipe suspend/resume.

Signed-off-by: Pavel Dubrova <pashadubrova@gmail.com>
---
 drivers/platform/msm/ipa/ipa_v3/ipa_utils.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/platform/msm/ipa/ipa_v3/ipa_utils.c b/drivers/platform/msm/ipa/ipa_v3/ipa_utils.c
index 6ef607e74cdf..dfb17dda7a1b 100644
--- a/drivers/platform/msm/ipa/ipa_v3/ipa_utils.c
+++ b/drivers/platform/msm/ipa/ipa_v3/ipa_utils.c
@@ -7845,11 +7845,6 @@ static int _ipa_suspend_resume_pipe(enum ipa_client_type client, bool suspend)
 	struct ipa3_ep_context *ep;
 	int res;
 
-	if (ipa3_ctx->ipa_hw_type < IPA_HW_v4_0) {
-		IPAERR("not supported\n");
-		return -EPERM;
-	}
-
 	ipa_ep_idx = ipa3_get_ep_mapping(client);
 	if (ipa_ep_idx < 0) {
 		IPADBG("client %d not configued\n", client);
-- 
2.48.1

